integration-cli/docker_api_inspect_test.go

```diff
@@ -42,3 +43,33 @@ func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
 		c.Fatalf("Path of `true` should not be converted to boolean `true` via JSON marshalling")
 	}
 }
+
+func (s *DockerSuite) TestInspectApiImageResponse(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "pull", "busybox")
+	out, _, err := runCommandWithOutput(runCmd)
+
+	if err != nil {
+		c.Fatalf("failed to pull image: %s, %v", out, err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "images", "-q", "busybox")
+	out, _, err = runCommandWithOutput(runCmd)
+
+	if err != nil {
+		c.Fatalf("failed to get image: %s, %v", out, err)
+	}
+
+	imageID := strings.TrimSpace(out)
+
+	endpoint := "/images/" + imageID + "/json"
+	status, body, err := sockRequest("GET", endpoint, nil)
+
+	c.Assert(status, check.Equals, http.StatusOK)
+	c.Assert(err, check.IsNil)
+
+	var inspectJSON types.ImageInspect
+	if err = json.Unmarshal(body, &inspectJSON); err != nil {
```

Comment:
I think it would be good to check the output JSON to make sure that the 'tags' field is there.